frozen-abi-macro/src/lib.rs

```diff
@@ -400,31 +482,39 @@ fn frozen_abi_enum_type(input: ItemEnum, expected_digest: &str) -> TokenStream {
 #[cfg(feature = "frozen-abi")]
 #[proc_macro_attribute]
 pub fn frozen_abi(attrs: TokenStream, item: TokenStream) -> TokenStream {
-    let mut expected_digest: Option<String> = None;
+    let mut api_expected_digest: Option<String> = None;
+    let mut abi_expected_digest: Option<String> = None;
+
     let attrs_parser = syn::meta::parser(|meta| {
         if meta.path.is_ident("digest") {
```

Comment:
How about deprecating `digest` as an attribute and having `api_digest` instead? It'll be clearer to downstream users what's really going on.

Reply:
We initially wanted this to be more of a drop-in update, but making things clear is probably preferable indeed.